x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts.tsx

```diff
@@ -37,6 +39,9 @@ export const AlertsSummaryContent = ({
   const { featureFlags } = usePluginConfig();
   const [isAlertFlyoutVisible, { toggle: toggleAlertFlyout }] = useBoolean(false);
   const { overrides } = useAssetDetailsRenderPropsContext();
+  const [isAlertSectionOpen, setIsAlertSectionOpen] =
```

Comment:
nit: This prop is not a boolean, we could name it `collapsibleStatus`, wdyt?

Reply:
Makes sense, renamed ✅